overlord/snapstate/aliases.go

```diff
@@ -132,27 +164,38 @@ func (m *SnapManager) doAlias(t *state.Task, _ *tomb.Tomb) error {
 		aliasStatuses = make(map[string]string)
 	}
 	var add []*backend.Alias
-	for alias := range toEnable {
+	var remove []*backend.Alias
+	for alias, newStatus := range toggles {
 		aliasApp := curInfo.Aliases[alias]
 		if aliasApp == nil {
-			return fmt.Errorf("cannot enable alias %q for %q, no such alias", alias, snapName)
+			return fmt.Errorf("cannot toggle alias %q for %q, no such alias", alias, snapName)
```

Comment:
tiny issue here is that (AFAICT) we're introducing the term "toggle" in a rarely-seen error message. I'd rather we used the same verbs here as elsewhere.

you could even do the "alias cannot be %s" thing if you didn't want the extra branch here